<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;单词索引器&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>651</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>导出文件</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>   单词</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>15</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>15</number>
       </property>
       <property name="text">
        <string>导入文件</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ExportFileName">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ImportFileName">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="SearchWord">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>651</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ShowLayout">
     <item row="0" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="ShowPanel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>330</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Helvetica Neue DeskInterface'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>673</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="ExportFile">
     <property name="title">
      <string>导出</string>
     </property>
     <addaction name="ExportStatic"/>
     <addaction name="ExportHash"/>
     <addaction name="ExportSort"/>
    </widget>
    <widget class="QMenu" name="Import">
     <property name="title">
      <string>导入</string>
     </property>
     <addaction name="ImportWordSource"/>
     <addaction name="ImportStatic"/>
     <addaction name="ImportHash"/>
    </widget>
    <addaction name="Import"/>
    <addaction name="ExportFile"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="Build">
    <property name="title">
     <string>建表</string>
    </property>
    <addaction name="BuildStatic"/>
    <addaction name="BuildSort"/>
    <addaction name="BuildHash"/>
   </widget>
   <widget class="QMenu" name="Traverse">
    <property name="title">
     <string>遍历</string>
    </property>
    <addaction name="TraverseStatic"/>
    <addaction name="TraverseSort"/>
    <addaction name="TraverseHash"/>
   </widget>
   <widget class="QMenu" name="Search">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="SearchStatic"/>
    <addaction name="SearchSort"/>
    <addaction name="SearchHash"/>
   </widget>
   <widget class="QMenu" name="Destroy">
    <property name="title">
     <string>删除</string>
    </property>
    <widget class="QMenu" name="DestroyHash">
     <property name="title">
      <string>哈希表</string>
     </property>
     <addaction name="DestroyWord"/>
     <addaction name="DestroyTable"/>
    </widget>
    <addaction name="DestroyStatic"/>
    <addaction name="DestroySort"/>
    <addaction name="DestroyHash"/>
   </widget>
   <widget class="QMenu" name="Analyze">
    <property name="title">
     <string>分析</string>
    </property>
    <addaction name="AnalyzeHash"/>
    <addaction name="AnalyzeTable"/>
   </widget>
   <widget class="QMenu" name="Help">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="HelpDeveloper"/>
    <addaction name="HelpAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="Build"/>
   <addaction name="Search"/>
   <addaction name="Traverse"/>
   <addaction name="Destroy"/>
   <addaction name="Analyze"/>
   <addaction name="Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="ExportStatic">
   <property name="text">
    <string>静态表</string>
   </property>
  </action>
  <action name="ExportHash">
   <property name="text">
    <string>哈希表</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="BuildSort">
   <property name="text">
    <string>有序表</string>
   </property>
  </action>
  <action name="BuildHash">
   <property name="text">
    <string>哈希表</string>
   </property>
  </action>
  <action name="TraverseStatic">
   <property name="text">
    <string>静态表</string>
   </property>
  </action>
  <action name="TraverseSort">
   <property name="text">
    <string>有序表</string>
   </property>
  </action>
  <action name="TraverseHash">
   <property name="text">
    <string>哈希表</string>
   </property>
  </action>
  <action name="SearchStatic">
   <property name="text">
    <string>静态表</string>
   </property>
  </action>
  <action name="SearchSort">
   <property name="text">
    <string>有序表</string>
   </property>
  </action>
  <action name="SearchHash">
   <property name="text">
    <string>哈希表</string>
   </property>
  </action>
  <action name="DestroyStatic">
   <property name="text">
    <string>静态表</string>
   </property>
  </action>
  <action name="DestroySort">
   <property name="text">
    <string>有序表</string>
   </property>
  </action>
  <action name="AnalyzeHash">
   <property name="text">
    <string>哈希分析</string>
   </property>
  </action>
  <action name="AnalyzeTable">
   <property name="text">
    <string>性能分析</string>
   </property>
  </action>
  <action name="DestroyWord">
   <property name="text">
    <string>删除单词</string>
   </property>
  </action>
  <action name="DestroyTable">
   <property name="text">
    <string>删除表</string>
   </property>
  </action>
  <action name="HelpDeveloper">
   <property name="text">
    <string>开发者</string>
   </property>
  </action>
  <action name="HelpAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="ImportWordSource">
   <property name="text">
    <string>从词源导入</string>
   </property>
  </action>
  <action name="ImportStatic">
   <property name="text">
    <string>静态表</string>
   </property>
  </action>
  <action name="ImportHash">
   <property name="text">
    <string>哈希表</string>
   </property>
  </action>
  <action name="BuildStatic">
   <property name="text">
    <string>静态表</string>
   </property>
  </action>
  <action name="ExportSort">
   <property name="text">
    <string>有序表</string>
   </property>
  </action>
  <action name="ImportSort">
   <property name="text">
    <string>有序表</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
